light-client/src/contracts.rs

```diff
@@ -12,6 +12,7 @@ pub fn trusted_store_contains_block_at_target_height(
     target_height: Height,
 ) -> bool {
     light_store.get(target_height, Status::Verified).is_some()
+        || light_store.get(target_height, Status::Trusted).is_some()
```

Comment:
I'm wondering if there are more places like this. reminded me a little of #419

Reply:
Yeah I thought I had caught them all in #419 but apparently not. Will go over the codebase again to make sure.